docs/api.md

```diff
@@ -67,9 +68,7 @@ localforage.getItem('somekey', function(err, value) {
     console.log(value);
 });
 
-Or, use `async`/`await`:
-
-```js
+// Or, use `async`/`await`:
```

Comment:
I'd rather these not be code comments but text, as they're instructions to the reader.